ProjFS.Mac/PrjFSKext/KauthHandlerTestable.hpp

```diff
@@ -1,7 +1,6 @@
 #include "public/PrjFSCommon.h"
 #include <sys/kernel_types.h>
-#include <sys/vnode.h>
```

Comment:
From now on, kext & unit tests must never `#include <sys/vnode.h>` directly, and always go via the `kernel-header-wrappers/vnode.h` because the kernel version of the file is mutually exclusive with the user space one, and we need the kernel one for a bunch of declarations.